app/dts/bindings/behaviors/zmk,behavior-tap-dance.yaml

```diff
@@ -13,4 +13,6 @@ properties:
     required: true
   tapping-term-ms:
     type: int
-    default: 200
\ No newline at end of file
+    default: 200
+  on-each-press:
```

Comment:
Not sure I love this naming... Brainstorming other ideas:

* `trigger-each-binding`
* `tap-every-binding`
* `tap-each`

Thoughts?